examples/using-guidebot-flags.md

```diff
@@ -44,44 +44,48 @@ However, you can use the switch case with the neat little `message.flags` concep
 
 ## Using with Arguments
 
-Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using the `message.flags` method. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
+Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using `message.flags`. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
 
 ```js
 exports.run = (client, message, args, level) => { // eslint-disable-line no-unused-vars 
   switch (message.flags[0]) {
     // This would be 'role -add'.
-    case ('add'): {
+    case 'add': {
       // Check if the message mentions a user.
-      if (message.mentions.users.size === 0) return message.reply('Please mention a user to give the role to.');
+      if (!message.mentions.users) return message.reply('Please mention a user to give the role to.');
```

Comment:
Collection is always [Truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy). You want to check size.

Reply:
Did a derp.